### PR TITLE
Fixed typos in README.md 

### PR DESCRIPTION
**Pull request by itsOmSarraf**
## Describe your changes 

- added fullstops and corrected the hyperlinks under 'Scan Periodically using Helm'
- changed the case of Windows, earlier it was written in the small case, now in the Title case.
- added full stops and formatted the hyperlinks under 'Scan Periodically using Helm'

## Screenshots - If Any (Optional)
before: 
image
after:
image

## This PR fixes: typos in README.md

* Resolved #

## Checklist before requesting a review

- [ ] My code follows the style guidelines of this project
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ kubescape scan --submit --enable-host-scan --verbose
 
 </br>
 
-### Please [star ⭐](https://github.com/kubescape/kubescape/stargazers) the repo if you want us to continue developing and improving Kubescape 😀
+### Please [star ⭐](https://github.com/kubescape/kubescape/stargazers) the repo if you want us to continue developing and improving Kubescape 😀.
```
Comment (dwertent) [inline README.md:54]: Remove `.`

File: README.md [hunk 2]
```diff
@@ -266,44 +266,44 @@ It is possible to run Kubescape offline!
...
+, helm chart [repo](https://github.com/armosec/armo-helm).
 
 # Integrations
 
 ## VS Code Extension 
 
 ![Visual Studio Marketplace Downloads](https://img.shields.io/visual-studio-marketplace/d/kubescape.kubescape?label=VScode) ![Open VSX](https://img.shields.io/open-vsx/dt/kubescape/kubescape?label=openVSX&color=yellowgreen)
 
-Scan the YAML files while writing them using the [vs code extension](https://github.com/armosec/vscode-kubescape/blob/master/README.md) 
+Scan the YAML files while writing them using the [vs code extension.](https://github.com/armosec/vscode-kubescape/blob/master/README.md)
```
Comment (dwertent) [inline README.md:302]:
```
Scan the YAML files while writing them using the [vs code extension.](https://github.com/armosec/vscode-kubescape/blob/master/README.md)
```
->
```
Scan the YAML files while writing them using the [vs code extension](https://github.com/armosec/vscode-kubescape/blob/master/README.md).
```

File: README.md [hunk 3]
```diff
@@ -266,44 +266,44 @@ It is possible to run Kubescape offline!
...
 
 ![Visual Studio Marketplace Downloads](https://img.shields.io/visual-studio-marketplace/d/kubescape.kubescape?label=VScode) ![Open VSX](https://img.shields.io/open-vsx/dt/kubescape/kubescape?label=openVSX&color=yellowgreen)
 
-Scan the YAML files while writing them using the [vs code extension](https://github.com/armosec/vscode-kubescape/blob/master/README.md) 
+Scan the YAML files while writing them using the [vs code extension.](https://github.com/armosec/vscode-kubescape/blob/master/README.md) 
 
 ## Lens Extension
 
-View Kubescape scan results directly in [Lens IDE](https://k8slens.dev/) using kubescape [Lens extension](https://github.com/armosec/lens-kubescape/blob/master/README.md)
+View Kubescape scan results directly in [Lens IDE](https://k8slens.dev/) using kubescape [Lens extension.](https://github.com/armosec/lens-kubescape/blob/master/README.md)
```
Comment (dwertent) [inline README.md:306]: `[Lens extension.](https://github.com/armosec/lens-kubescape/blob/master/README.md)` -> `[Lens extension](https://github.com/armosec/lens-kubescape/blob/master/README.md).`

**Review by dwertent — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (itsOmSarraf) [reply README.md:54]: Should I make a new pull request in order to resolve these changes ?

File: README.md (hunk 2 above)
Comment (itsOmSarraf) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (itsOmSarraf) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (itsOmSarraf) [reply README.md]: done

**Pull request closed without merging**